readthedocs/core/views/__init__.py

```diff
@@ -27,14 +28,27 @@ class HealthCheckView(CDNCacheControlMixin, View):
     # but it's useful to be careful here in case of a misconfiguration.
     cache_response = False
 
-    def get(self, request, *args, **kwargs):
+    def get(self, request, *_, **__):
         return JsonResponse({'status': 200}, status=200)
 
 
 class HomepageView(TemplateView):
 
+    """
+    Conditionally show the home page or redirevt to the login page.
```

Comment:
```suggestion
    Conditionally show the home page or redirect to the login page.
```

I can see the new keyboard here 😄